### PR TITLE
docs: mention docusaurus.community site in resources

### LINKED ISSUE #8939: Mention docusaurus.community docs under Awesome Resources
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#issues).

### Description

@thadguidry -
> WHY is https://docusaurus.community/ not even given a passing mention under Awesome Resources or Support or somewhere else?

@homotechsual -
> homotechsual — 2023-4-27 at 9:10 PM
> Because I keep forgetting to PR it into that page 🤣
> Once I finish v2 of the plugin directory it might even move over to docusaurus.io for that part.

Now you have an issue to assign for yourself for a future PR.

### Self-service

- [x] I'd be willing to address this documentation request myself.

### PR DESCRIPTION
**Pull request by thadguidry**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

https://Docusaurus.community site should be mentioned under Community.

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

## Related issues/PRs

Fixes #8939

### CODE REVIEW

**Comment by thadguidry** [comment]
I have no idea how to fix the info admonition autocorrecting!

It's supposed to look like this:

```
:::info
We might allow direct links to be added in our official documentation that point to the community documentation, on a case by case basis, and as the community requests that such links to articles are useful.

We are also contemplating further integration into the official documentation at some point in the future.
:::

So that it's not super bolded as an info title for the whole 1st sentence!

File: website/community/2-community-docs.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+(extends the [Official Docs](/docs))
+
+The **[Docusaurus.community](https://docusaurus.community)** site can be used for extending the official documentation with more complex examples and full articles on best practices that provide solutions to problems the community often faces.
+
+(It is also a full React application just like our main site. So customization and experimentation examples are perfectly suited to be hosted there.)
+
+:::info We might allow direct links to be added in our official documentation that point to the community documentation, on a case by case basis, and as the community requests that such links to articles are useful.
+
+We are also contemplating further integration into the official documentation at some point in the future. :::
```
Comment (Josh-Cena) [inline website/community/2-community-docs.mdx:17]:
```suggestion
# Community Documentation

The **[Docusaurus.community](https://docusaurus.community)** site extends the [official docs](/docs) with more complex examples and full articles that recommend best practices and provide solutions to common problems.

It is also a full Docusaurus application just like our main site, so customization and experimentation examples are perfectly suited to be hosted there.

:::info

We might allow direct links to be added in our official documentation that point to the community documentation, on a case-by-case basis, if the community finds such links to articles useful.

We are also contemplating further integration into the official documentation at some point in the future.

:::
```

Not sure why it needs a single H2 heading.

**Review by Josh-Cena — COMMENTED** [review]
I don't think this needs an entire page. Just putting it in "resources" is good enough.

**Comment by Josh-Cena** [comment]
> I have no idea how to fix the info admonition autocorrecting!

@thadguidry This is because of Prettier autoformatting. Please see https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier

**Comment by thadguidry** [comment]
> > I have no idea how to fix the info admonition autocorrecting!
> 
> @thadguidry This is because of Prettier autoformatting. Please see https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier

Prettier extension is disabled in my VSCode and yet something is still autocorrecting the info admonition formatting once I click on commit (but not push yet).

**Comment by Josh-Cena** [comment]
> Prettier extension is disabled in my VSCode and yet something is still autocorrecting the info admonition formatting once I click on commit (but not push yet).

We have a husky hook that automatically runs Prettier on pre-commit.

File: website/community/2-resources.mdx [hunk 2]
```diff
@@ -2,6 +2,16 @@
 
 A curated list of interesting Docusaurus community projects.
 
+## Community Documentation
```
Comment (Josh-Cena) [inline website/community/2-resources.mdx:5]:
```suggestion
## Community documentation {#community-documentation}
```

**Comment by Josh-Cena** [comment]
@thadguidry Please, please read https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier — even if you hack around the pre-commit hook, this style will be rejected by the CI. All you need to do is add line breaks around the `:::` fences.

**Review by slorber — APPROVED** [review]
Thanks 👍 

Ok to add this once you are ready with the text

**Pull request merged**